website/blog/2021-05-25-inking.md

```diff
@@ -0,0 +1,412 @@
+---
+title: "Add inking to your React Native for Windows application"
+author: Matteo Pagani
+authorURL: https://twitter.com/qmatteoq
+description: "How to use add inking to a React Native for Windows application"
+is_blog: true
+published_date: May 25, 2021
+---
+
+Windows 10 is one of the most powerful platforms on the market when it comes to supporting digital inking experiences. As such, the Universal Windows Platform comes with a dedicated series of controls that you can add to your application to enable inking experiences. Thanks to these controls, you can easily implements features like capturing signatures, text and shape recognition, etc.
+
+The starting point is the `InkCanvas` control which provides a canvas that is able to react to inking interactions. However, being a specific Windows 10 control, it isn't automatically exposed by React Native, since we don't have an equivalent control on other platforms. As such, we'll have to wrap it with a [native UI component](https://microsoft.github.io/react-native-windows/docs/view-managers), which is a feature provided by React Native which enables to expose native UI controls for Windows, Android or iOS to the JavaScript layer using JSX. 
```

Comment:
```suggestion
The starting point is the XAML [`InkCanvas`](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.InkCanvas) control which provides a canvas that handles inking interactions. However, being a native XAML control, it isn't automatically exposed by React Native, since we don't have an equivalent control on other platforms. As such, we'll have to wrap it with a [native UI component](https://microsoft.github.io/react-native-windows/docs/view-managers), which is a feature provided by React Native which enables to expose native UI controls for Windows, Android or iOS to the JavaScript layer using JSX. 
```